### PR TITLE
Add Batwoman episodes to the list

### PR DESCRIPTION
**Pull request by neil-menghani**
I have have added functionality to track Batwoman episodes in the list of episodes. Please let me know if you have any issues with these changes, and I can make adjustments.

### CODE REVIEW

**Comment by neil-menghani** [comment]
I have also tested the flask app and it works properly. I have chosen a color that works with Batwoman (dark purple) which does not also clash with Vixen.

**Comment by neil-menghani** [comment]
Forgot to mention that I added Black Lightning to the Readme list of shows tracked as well. This change is redundant with #120 so feel free to use that adjust my pull request accordingly if that pull request is accepted

File: ordering/utils.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from bs4 import BeautifulSoup
 
 from . import app
-from .constants import ARROW, CONSTANTINE, FLASH, FREEDOM_FIGHTERS, LEGENDS_OF_TOMORROW, SUPERGIRL, WIKIPEDIA
+from .constants import ARROW, CONSTANTINE, FLASH, FREEDOM_FIGHTERS, LEGENDS_OF_TOMORROW, SUPERGIRL, BATWOMAN, WIKIPEDIA
```
Comment (AceFire6) [inline ordering/utils.py]:
```suggestion
from .constants import ARROW, BATWOMAN, CONSTANTINE, FLASH, FREEDOM_FIGHTERS, LEGENDS_OF_TOMORROW, SUPERGIRL, WIKIPEDIA
```

**Comment by neil-menghani** [comment]
Done, the constants are in alphabetical order now. Any other feedback?

**Comment by AceFire6** [comment]
Looks good to me 😄 Thanks for making the pull request!

**Pull request merged**